src/styles/price.css

```diff
@@ -0,0 +1,19 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
```

Comment:
The block should not have margin and position at all as per BEM. Please remove margin from here.